module/VuFind/src/VuFind/AjaxHandler/DoiLookupFactory.php

```diff
@@ -68,12 +68,11 @@ public function __invoke(
             throw new \Exception('Unexpected options passed to factory.');
         }
         $config = $container->get(\VuFind\Config\PluginManager::class)
-            ->get('config');
-        $pluginManager = $container->get(\VuFind\DoiLinker\PluginManager::class);
+            ->get('config')->toArray();
```

Comment:
Is there a reason to send the whole configuration, or would it be cleaner to send just the DOI section? (I'm fine with sending it all if you think it might improve future flexibility, but it would be less verbose to send the relevant section otherwise).

Reply:
It would be cleaner, but I did it this way to avoid having to change the signature if something else from the config is needed in future, like happened with BrowZine.php here, or in a derived class.